_layouts/text-home.html

```diff
@@ -0,0 +1,51 @@
+<!doctype html>
+{% include head.html %}
+
+<body>
+    {% include header-extended.html %}
+
+    <div class="usa-section">
+        <div class="grid-container">
+            <div class="grid-row grid-gap">
+                <div class="usa-layout-docs__sidenav desktop:grid-col-3">
+                    <nav aria-label="Secondary navigation">
+                        {% include nav.html %}
+                    </nav>
+                </div>
+                
+                <main class="usa-layout-docs__main desktop:grid-col-9 usa-prose usa-layout-docs" id="main-content">
+                    <div class="grid-container">
+                        <h1 class="font-heading-2xl">{{ page.title }}</h1>
```

Comment:
Try removing top margin on this heading and see if you like the alignment with the nav better.